apps/server/src/controllers/dictionaryController.ts

```diff
@@ -133,19 +135,56 @@ export const getSchemaField = async (
 	const dictionary = await dictionaryService.getOneById(dictId);
 	const formattedDictionary = replaceReferences(dictionary);
 
-	const schema = formattedDictionary.schemas.find((schema) => schema.name === schemaName);
+	const schema = formattedDictionary.schemas.find((schema: { name: string }) => schema.name === schemaName);
 
 	if (!schema) {
 		throw new NotFoundError(
 			`Dictionary '${dictionary.name} ${dictionary.version}' does not have a schema named '${schemaName}'`,
 		);
 	}
 
-	const field = schema.fields.find((field) => field.name === fieldName);
+	const field = schema.fields.find((field: { name: string }) => field.name === fieldName);
 	if (!field) {
 		throw new NotFoundError(
 			`Schema '${schemaName}' from Dictionary '${dictionary.name} ${dictionary.version}' does not have a field named '${fieldName}'`,
 		);
 	}
 	res.send(field);
 };
+
+export const downloadTemplates = async (req: Request<{}, {}, {}, { name: string; version: string }>, res: Response) => {
+	const { name, version } = req.query;
+
+	if (!name || !version) {
+		throw new BadRequestError('Missing dictionary name or version in query.');
+	}
+
+	const dictionary = await dictionaryService.downloadDictionaryByNameAndVersion(name, version);
```

Comment:
Need to handle error cases where dictionary is not found, or if the DB fetch fails.